release-notes/2025-12/README.md

```diff
@@ -0,0 +1,164 @@
+# SystemLink Enterprise release 2025-12 Release Notes
+
+The 2025-12 release for SystemLink Enterprise has been published to
+<https://downloads.artifacts.ni.com>. This update includes new features, bug
+fixes, and security updates. Work with your account representative to obtain
+credentials to access these artifacts. If you are not upgrading from the
+previous release, refer to past release notes to ensure you have addressed all
+required configuration changes.
+
+## Upgrading from the release year-release-month to the release year-release-month
+
+<!-- Optional section to include comments and instructions needed to successfully upgrade from the previous release to the current release. If the only changes needed are already captured in Helm Chart Breaking Changes, this section is not needed. -->
+
+## New Features and Behavior changes
+
+- Behavior change or new feature description
+
+- Behavior change or new feature description
+
+## Helm Chart Breaking Changes
+
+- dataframeservice
```

Comment:
We should reference the version here. Refer to past release notes for how this is formatted.